src/components/Process/Create/Sidebar/ExtraConfig.tsx

```diff
@@ -3,19 +3,75 @@ import {
   FormControl,
   FormErrorMessage,
   FormLabel,
-  NumberInput,
-  NumberInputField,
+  HStack,
+  Icon,
+  IconButton,
   Spinner,
+  Text,
   VStack,
 } from '@chakra-ui/react'
 import { chakraComponents, Select } from 'chakra-react-select'
 import { useEffect, useState } from 'react'
 import { Controller, useFormContext } from 'react-hook-form'
 import { Trans, useTranslation } from 'react-i18next'
+import { LuMinus, LuPlus } from 'react-icons/lu'
 import { useGroups } from '~src/queries/groups'
 import { useValidations } from '~utils/validation'
 import { CensusTypes } from './CensusCreation'
 
+const VoteChangesStepper = ({ value, onChange, isDisabled }) => {
+  const handleDecrement = () => {
+    const newValue = Math.max(1, (value || 1) - 1)
+    onChange(newValue)
+  }
+
+  const handleIncrement = () => {
+    const newValue = Math.min(10, (value || 1) + 1)
+    onChange(newValue)
+  }
+
+  return (
+    <HStack spacing={0} border='1px' borderColor='gray.200' borderRadius='md' overflow='hidden' w='full'>
+      <IconButton
+        icon={<Icon as={LuMinus} />}
+        size='md'
+        isDisabled={isDisabled || (value || 1) <= 1}
+        onClick={handleDecrement}
+        variant='ghost'
+        borderRadius='none'
+        aria-label='Decrease vote changes'
+        flex='1'
+        minW='48px'
+      />
+      <Text
+        flex='2'
+        textAlign='center'
+        py={3}
+        px={4}
+        borderLeft='1px'
+        borderRight='1px'
+        borderColor='gray.200'
```

Comment:
Colors (in general) cannot be hardcoded, otherwise it's going to look bad on some of the themes